tfjs-backend-wasm/src/backend_wasm.ts

```diff
@@ -44,6 +44,12 @@ export class BackendWasm extends KernelBackend {
   constructor(public wasm: BackendWasmModule) {
     super();
     this.wasm.tfjs.init();
+    // Register the used thread pool size flag. Done it here to avoid circular
+    // import.
+    env().registerFlag('WASM_THREAD_POOL_SIZE', () => {
```

Comment:
Usually, we register flag in `flags_wasm.ts`. And call `env().set('WASM_THREAD_POOL_SIZE', this.getThreadPoolSize());`. But I am not sure if it is always the rules.

Reply:
Currently backend_wasm.ts  deppends on flags_wasm.ts.
If we put those code in flags_wasm.ts, flags_wasm.ts will also depends on  backend_wasm.ts, the test case will pass, but the yarn lint will fail due to  circular imports.